### PR TITLE
Group by location and expose it to Flask route

### PR DESCRIPTION
**Pull request by ashwinvis**

### CODE REVIEW

File: server/routes.py [hunk 1]
```diff
@@ -56,6 +59,7 @@ def analyse():
         while count_report_to_analyse() > 0:
             map_calculate(batch_size)
 
+        group_reports_by_location()
```
Comment (marcusthorstrom) [inline server/routes.py:62]: Should we do batch-processing before http response? Maybe respond and start a backgroud job with processing?

**Review by marcusthorstrom — APPROVED** [review]
Just a minor thought

File: server/routes.py (hunk 1 above)
Comment (ashwinvis) [reply server/routes.py:62]: You mean like a cron job?

Comment (ashwinvis) [reply server/routes.py:62]: Just run `analysis/scripts/05_script_analysis.py` then

**Pull request merged**